contents/product-engineers/nps-vs-csat-vs-ces.md

```diff
@@ -0,0 +1,202 @@
+  - Product engineers
+  - Surveys
+---
+
+Here's the short version of what you need to know about NPS, CSAT, and CES:
+
+- **NPS** is for measuring long-term user sentiment about your product.
+- **CSAT** tells you how users feel about your product _right now_.
+- **CES** is for understanding if your product is difficult to use.
+
+And now for the longer version…
+
+In this post, we'll go into the specifics of each [survey](/surveys) and how to use them in your SaaS app.
+
+## What is NPS?
+
+Net promoter score (NPS) measures user loyalty by asking them how likely they are to recommend you to a friend or coworker. 
+
+Users submit their responses on a scale from 0 (not likely at all) to 10 (extremely likely).
+
+![NPS survey](../images/blog/nps-csat-ces/nps-survey.png)
+
+NPS is great as it gives you insight into who your loyal customers are and which are at risk of churning.
+
+### How to calculate NPS score
+
+NPS survey responses are grouped into three categories:
+
+- **Promoters** – people who respond with 9 or 10. They're extremely happy with your product.
+- **Passives** – people who respond with 7 or 8. They aren't unhappy, but they're not overly excited either.
+- **Detractors** – people who respond with 6 or below. These people have had a bad experience and are likely to complain to others.
+
+![NPS scale](../images/blog/nps-csat-ces/nps-score.png)
+
+Your NPS score is the `% of promoters - % of detractors`. The scores range between -100 (100% negative) and +100 (100% positive).
+
+For example, let's say you survey `100` people. `70` people give a promoter score and `20` people give a detractor score. Your NPS would be `70% - 20% = 50%`.
+
+### What is a good NPS score for SaaS?
+
+Anything above 0 is good, above 20 is great, and above 50 is amazing. According to [Satmetrix](https://www.satmetrix.com/wp-content/uploads/2023/07/NICE-Net-Promoter-Benchmarks-2023.pdf), the average NPS score for SaaS apps is 27.
+
+Below is a table with a benchmark of NPS scores from well-known SaaS products.
+
+| Product | NPS score |
+|--------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| [Snowflake](https://www.snowflake.com/blog/customer-experience-report-2022/) | 72 |
+| [Square](https://d1g145x70srn7h.cloudfront.net/documents/investor-relations/presentations/05-11-2018-overview.pdf) | 70 |
+| [Zoom](https://blog.zoom.us/zooms-nps-dominates-video-conferencing/) | 69 | 
+| [DocuSign](https://www.docusign.com/why-docusign) | 66 |
+| [Zapier](https://www.comparably.com/brands/zapier) | 64 |
+| [Slack](https://www.comparably.com/brands/slack) | 53 |
+| [Mailchimp](https://www.comparably.com/brands/mailchimp) | 47 |
+| [NetFlix](https://www.comparably.com/brands/netflix) | 46 |
+| [Twilio](https://www.comparably.com/brands/twilio) | 34 |
+| [Dropbox](https://www.comparably.com/brands/dropbox) | 27 | 
```

Comment:
Would love feedback on this table. Is it weird that a lot of the links point to comparably? Also, I dont include a table like this for CSAT or CES since I couldn't find data

Reply:
I think the table is good 👍